src/components/month/calendarMonthView.component.ts

```diff
@@ -213,6 +218,13 @@ export class CalendarMonthViewComponent implements OnChanges, OnInit, OnDestroy
   /**
    * @hidden
    */
+  getDays(): WeekDay[] {
+    return this.view.days.filter(d => !this.excludeDays.some(ex => ex === d.date.getDay()));
```

Comment:
This logic should be done once in the `calendar-utils` module on the `getMonthView` function